Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -53,7 +53,7 @@ do
 	case "${option}"
 	in
 		n) NUM_VEHICLES=${OPTARG};;
-		m) VEHICLE_MODEL=${OPTARG};;
+		m) VEHICLE_MODEL=${OPTARG:=iris};;
```

Comment:
Could you just add it as it was done previously? 

This way the model will not be iris if there is no -m flag specified

Reply:
I'm not sure I understand what you mean?

Reply:
@Jaeyoung-Lim I think the latest commit is what you were looking for